service/history/ndc/workflow_resetter.go

```diff
@@ -733,6 +748,15 @@ func (r *workflowResetterImpl) reapplyContinueAsNewWorkflowEvents(
 			return "", err
 		}
 	}
+	childrenInitializedAfterReset := make(map[string]bool)
+	for _, event := range childInitEventsAfterReset {
+		attr := event.GetStartChildWorkflowExecutionInitiatedEventAttributes()
+		childID := fmt.Sprintf("%s:%s", attr.GetWorkflowType().Name, attr.GetWorkflowId())
+		childrenInitializedAfterReset[childID] = true
+	}
+	if len(childrenInitializedAfterReset) > 0 {
+		resetMutableState.SetChildrenInitializedPostResetPoint(childrenInitializedAfterReset)
```

Comment:
are we carrying over this field upon continueAsNew? 
Same Q for other fields like ResetRunID and original executionRunID. (maybe we don't need to carry over them, just checking.)

Reply:
We need to carry them over since this data is used at the time of starting the child. And it's also used while accepting the results from the child.

Reply:
Synced offline. There will be a separate PR for carrying over the information.